client/payment-details/summary/index.tsx

```diff
@@ -246,13 +278,69 @@ const PaymentDetailsSummary: React.FC< PaymentDetailsSummaryProps > = ( {
 									placeholder="Fee amount"
 								>
 									{ `${ __(
-										'Fee',
+										'Fees',
 										'woocommerce-payments'
 									) }: ` }
 									{ formatCurrency(
 										-balance.fee,
 										balance.currency
 									) }
+									{ disputeFee && (
+										<ClickTooltip
+											className="payment-details-summary__breakdown__fee-tooltip"
+											buttonIcon={ <HelpOutlineIcon /> }
+											buttonLabel={ __(
+												'Fee breakdown',
+												'woocommerce-payments'
+											) }
+											content={
+												<>
+													<Flex>
+														<label>
+															{ __(
+																'Transaction fee',
+																'woocommerce-payments'
+															) }
+														</label>
+														<span aria-label="Transaction fee">
+															{ formatCurrency(
+																transactionFee.fee,
+																transactionFee.currency
+															) }
+														</span>
+													</Flex>
+													<Flex>
+														<label>
+															{ __(
+																'Dispute fee',
+																'woocommerce-payments'
+															) }
+														</label>
+														<span aria-label="Dispute fee">
+															{ formatCurrency(
+																disputeFee.fee,
+																disputeFee.currency
+															) }
+														</span>
+													</Flex>
+													<Flex>
+														<label>
+															{ __(
+																'Total fees',
+																'woocommerce-payments'
+															) }
+														</label>
+														<span aria-label="Total fees">
+															{ formatCurrency(
+																balance.fee,
+																balance.currency
+															) }
+														</span>
+													</Flex>
+												</>
+											}
+										/>
+									) }
```

Comment:
> What's the $33.08 headline number in timeline 🤔?
This I can answer: $18.08 USD disputed amount + $15 USD dispute fee.

Although, the current state of the UI doesn't make this clear. This could be moved to a new issue

Reply:
Sounds good to me – we can iterate on these details over time.